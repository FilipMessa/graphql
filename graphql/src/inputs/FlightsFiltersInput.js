// @flow

import { GraphQLInputObjectType, GraphQLInt } from 'graphql';

export default new GraphQLInputObjectType({
  name: 'FlightsFiltersInput',
  fields: {
    maxStopovers: {
      type: GraphQLInt,
    },
  },
});
