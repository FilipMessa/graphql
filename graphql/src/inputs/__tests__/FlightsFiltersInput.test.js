// @flow

import FlightsFiltersInput from '../../flight/types/inputs/FlightsFiltersInput';

it('FlightsFiltersInput type should have valid fields', () => {
  expect(FlightsFiltersInput.getFields()).toMatchSnapshot();
});
