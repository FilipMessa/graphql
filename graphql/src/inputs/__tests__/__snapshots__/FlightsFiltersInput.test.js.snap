// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FlightsFiltersInput type should have valid fields 1`] = `
Object {
  "maxStopovers": Object {
    "name": "maxStopovers",
    "type": "Int",
  },
}
`;
